doc/services/storage/secure_storage/index.rst

```diff
@@ -100,7 +100,7 @@ ITS API
 Zephyr's implementation of the ITS API
 (:kconfig:option:`CONFIG_SECURE_STORAGE_ITS_IMPLEMENTATION_ZEPHYR`)
 makes use of the ITS transform and store modules, which can be configured and customized separately.
-Have a look at the ``CONFIG_SECURE_STORAGE_ITS_(STORE|TRANSFORM)_.*_CUSTOM``
+Have a look at the :kconfig:option:`CONFIG_SECURE_STORAGE_ITS_(STORE|TRANSFORM)_.*_CUSTOM`
```

Comment:
given it's not supported at the moment, it's probably better to drop this change? If we are to add support for regex, my guess is that it would be using a different role as `:kconfig:option:` is really meant to map to a single option.